core/state_transition.go

```diff
@@ -432,19 +432,28 @@ func (st *StateTransition) innerTransitionDb() (*ExecutionResult, error) {
 		effectiveTip = cmath.BigMin(st.gasTipCap, new(big.Int).Sub(st.gasFeeCap, st.evm.Context.BaseFee))
 	}
 
+	gasUsed := new(big.Int).SetUint64(st.gasUsed())
+	kromaConfig := st.evm.ChainConfig().Kroma
+	blockNum := st.evm.Context.BlockNumber.Uint64()
```

Comment:
Strictly speaking, this is only used when `kromaConfig != nil` but you don't have to follow this. because it's too minor